test/unit/sfdxCommand.test.ts

```diff
@@ -108,10 +112,10 @@ let jsonToStdout: boolean;
 async function mockStdout(test: (outLines: string[]) => Promise<void>) {
   const oldStdoutWriter = process.stdout.write.bind(process.stdout);
   const lines: string[] = [];
-  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-  // @ts-ignore
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore TODO: why isn't `create` invokable?!
```

Comment:
Is this TODO comment accurate?

Reply:
no, that was copy pasted from the other one.